### PR TITLE
Splunk content fixes

### PR DESCRIPTION
**Pull request by David-BMS**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/37361

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/SplunkPy/ReleaseNotes/2_1_1.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+
+#### Mappers
+##### Splunk - Notable Generic Outgoing Mapper
+- Changed to from version 6.2
```
Comment (roysagi) [inline Packs/SplunkPy/ReleaseNotes/2_1_1.md:4]:
```suggestion
- You can now only use this mapper from Cortex XSOAR version 6.2 and above.
```

File: Packs/SplunkPy/ReleaseNotes/2_1_1.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+
+#### Playbooks
+##### Splunk Generic
+- Fixed playbook description and email task.
```
Comment (roysagi) [inline Packs/SplunkPy/ReleaseNotes/2_1_1.md:8]:
```suggestion
- Documentation and metadata improvements.
- Fixed an issue where the Email task did not work as expected.
```

File: Packs/SplunkPy/Playbooks/playbook-Splunk_Generic.yml [hunk 3]
```diff
@@ -52,7 +52,7 @@ inputs:
   playbookInputQuery:
   required: false
   value:
-    simple: 'Adjustment/Exclusion for offense '
+    simple: 'Adjustment/Exclusion for notable '
```
Comment (yaakovi) [inline Packs/SplunkPy/Playbooks/playbook-Splunk_Generic.yml:55]: is the space at the end supposed to be here or is that a bug?

Comment (David-BMS) [reply Packs/SplunkPy/Playbooks/playbook-Splunk_Generic.yml:55]: Supposed

**Pull request merged**